### PR TITLE
fix for error C2440:

### PR DESCRIPTION
**Pull request by gleeda**
'return': cannot convert from 'initializer list' to 'std::tuple<void *, size_t>'

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 using namespace std;
 
+typedef tuple<void*, size_t> MyTuple;
```
Comment (JLospinoso) [inline main.cpp]: Please put this typedef in the unnamed namespace below

File: main.cpp [hunk 2]
```diff
@@ -65,7 +68,9 @@ tuple<void*, size_t> allocate_pic(const string& filename) {
   DWORD old_protection;
   auto prot_result = VirtualProtectEx(GetCurrentProcess(), pic, pic_size, PAGE_EXECUTE_READ, &old_protection);
   if (!prot_result) throw runtime_error("[-] Couldn't VirtualProtectEx: " + GetLastError());
-  return { pic, pic_size };
+  MyTuple t1((void*) pic, (size_t) pic_size );
```
Comment (JLospinoso) [inline main.cpp]: Please use static_cast<void*> / static_cast<size_t> and just return as an rvalue:

```cpp
return MyTuple(static_cast<void*>(pic), static_cast<size_t>(pic_size));
```

**Review by JLospinoso — CHANGES_REQUESTED** [review]
Thanks so much for the pull request! Please make the suggested edits and I'll merge this in.

**Comment by JLospinoso** [comment]
Also, what compiler are you using? VS2015 didn't have a problem with the uniform initialization to Tuple.

**Comment by gleeda** [comment]
Sure, I can do that.  I have the express version of VS2015.  Not sure why it would have given me an issue, though, if you had none :-)

**Comment by JLospinoso** [comment]
@gleeda Interesting! Yeah it's definitely not supported in the C++11 standard (oddly) to use uniform initialization to Tuple. Not sure if an updated version of VS supports?

Anyway, its a good fix. I'll merge it in when you've got the tweaks complete.

**Review by JLospinoso — APPROVED** [review]

**Comment by JLospinoso** [comment]
@gleeda Thanks!

**Comment by gleeda** [comment]
No problem!  :-)

**Pull request merged**